src/wp-includes/rest-api/endpoints/class-wp-rest-widgets-controller.php

```diff
@@ -690,9 +690,8 @@ public function prepare_item_for_response( $item, $request ) {
 			'instance'      => null,
 		);
 
-		if (
-			rest_is_field_included( 'rendered', $fields ) &&
-			'wp_inactive_widgets' !== $sidebar_id
+		if ( 'wp_inactive_widgets' !== $sidebar_id
+			&& rest_is_field_included( 'rendered', $fields )
```

Comment:
iirc this one is wrong according to WPCS.

```suggestion
		if ( 
			'wp_inactive_widgets' !== $sidebar_id &&
			rest_is_field_included( 'rendered', $fields )
```

Reply:
Agreed ☝️

Reply:
@apermo, I’m open to changing this; I just couldn’t find any mention of it in WPCS. Additionally, there are over 100 examples in the codebase where `if()` blocks are formatted like in the original version.
I’d appreciate it if you could point me to the exact code example you’re referring to so I can learn more about it. Thank you.